tracing/tracing-test.sh

```diff
@@ -256,7 +261,16 @@ run_trace_forwarder()
 	source "$HOME/.cargo/env"
 	command -v "$forwarder_binary_name" &>/dev/null || (cd "$forwarder_dir" && cargo install --path .)
 
-	tmux new-session -d -s "$KATA_TMUX_FORWARDER_SESSION" "$forwarder_binary_name -l trace"
+	if [ $KATA_HYPERVISOR = "qemu" ]; then
```

Comment:
I suggest that you initialize the `KATA_HYPERVISOR` alongside the other global variables so that bash doesn't complain about it being non-set.